core/src/tvu.rs

```diff
@@ -298,7 +298,10 @@ impl Tvu {
         let (cost_update_sender, cost_update_receiver) = unbounded();
         let (drop_bank_sender, drop_bank_receiver) = unbounded();
         let (voting_sender, voting_receiver) = unbounded();
-        let (bls_sender, bls_receiver) = unbounded();
+        // The BLS sender channel should be mostly used during standstill handling,
+        // there could be 10s/400ms = 25 slots, <=5 votes and <=5 certificates per slot,
+        // we cap the channel at 512 to give some headroom.
```

Comment:
thanks for the comment!

Reply:
But if the network is in a partition where everyone is in a partition with <60% stake, then we shouldn't have many votes and certs constructed during the partition? If one partition is > 60% stake, then others should be able to observe new finalized cert and continue during recovery.